### ISSUE TITLE
Bulk edit authorization problem

### ISSUE THREAD

**Issue by elreydetoda** [snipe/snipe-it#6370]
#### Please confirm you have done the following before posting your bug report:

- [x] I have enabled debug mode 
- [x] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**
I can successfully import my users into snipe-it with the LDAP sync, but then when I got to assign them to groups it doesn't propagate permissions properly to them. When using a local account though all permissions are properly propagated.

**To Reproduce**
1. Go to 'Sync AD Users'
2. Login, successfully, as synced ad user
3. See 2 error after login from AD user (view in error section)

**Expected behavior**
For this user to see all super admin permission level item.

**Screenshots**
- Can't view anything with AD user: pic00
![2018-10-25-22 09 41_358x485](https://user-images.githubusercontent.com/10230166/47541111-10a1a580-d8a6-11e8-9785-88f5bed6b9b4.png)
- groups assigned to AD user
![2018-10-25-22 10 15_443x51](https://user-images.githubusercontent.com/10230166/47541602-d2a58100-d8a7-11e8-950b-06935a602e77.png)
- same view with locally provisioned user with same groups
![2018-10-25-22 40 17_297x736](https://user-images.githubusercontent.com/10230166/47541615-e224ca00-d8a7-11e8-8f94-ecab929fac98.png)

**Server (please complete the following information):**
 - Snipe-IT Version: v4.6.5
 - OS: [e.g. Ubuntu, CentOS]: ubuntu hosting snipe-it docker container
 - Web Server: [e.g. Apache, IIS]: in docker container
 - PHP Version: in docker container

**Desktop (please complete the following information):**
 - OS: [e.g. iOS]: windows server 16
 - Browser [e.g. chrome, safari]: chrome and firefox
 - Version [e.g. 22]: 69.0.3497.100 and 63.0

**Error Messages**
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error
- If a stacktrace is provided in the error, include that too.
```
This action is unauthorized.
/var/www/html/vendor/laravel/framework/src/Illuminate/Auth/Access/HandlesAuthorization.php#28
Illuminate\Auth\Access\AuthorizationException
     */
    protected function deny($message = 'This action is unauthorized.')
    {
        throw new AuthorizationException($message);
    }
}
```

- Any errors that appear in your browser's error console.
```
Failed to load resource: the server https://snipeit.website/api/v1/reports/activity?target_id=161&target_type=User&order=desc&order=desc&offset=0&limit=20
responded with a status of 500 (Internal Server Error)
```
- Confirm whether the error is reproducible on the demo: https://snipeitapp.com/demo.
  - n/a (doesn't have ldap)
- Include any additional information you can find in `storage/logs` and your webserver's logs.
[laravel.log](https://github.com/snipe/snipe-it/files/2518089/laravel.log)

**Additional context**
- Is this a fresh install or an upgrade? fresh install
- What OS and web server you're running Snipe-IT on: docker container defaults
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc): docker
- Include what you've done so far in the installation, and if you got any error messages along the way.: I have everything installed and everything works except for the ldap users permissions. 
- Indicate whether or not you've manually edited any data directly in the database: no

To clarify I can login with the ldap users, so it isn't an authentication problem (from what I can tell). I would think that would be an authorization problem, the user can authenticate but can't access what they need to even though they have the permission.

**Comment by jwhulette** [comment]
@elreydetoda When importing users, they are not assigned any permissions. The LDAP group does not translate into a SnipeIT group. You would have to manually assign SnipeIT groups to users that are synced with LDAP.

**Comment by elreydetoda** [comment]
Yes, I did that manually after I synced the AD users, and the screenshot that shows `groups` are the groups that one of my ad users has. Which are the same groups assigned to the local user that is properly displaying all the different admin options.

**Comment by jwhulette** [comment]
Ok, to clarify, you sync the AD user, you manually add the proper groups to the AD user but when AD user signs in they are not seeing the options the group settings should give them?

**Comment by elreydetoda** [comment]
Correct, sorry I was trying to explain that clearer in the initial post but wasn't very good at it...

**Comment by elreydetoda** [comment]
I am pretty sure that it has something to do with the api call that I mention above.
`https://snipeit.website/api/v1/reports/activity?target_id=` whenever I created an api token and did this with user ids. It returned back all of my users, ok well maybe not all my users but more than just the user_id I was searching for.

**Comment by jwhulette** [comment]
@elreydetoda I can't seem to reproduce. I synced an AD user. In an incognito window, log in as the AD user into Snipe-IT. It showed the user limited as it should. Then, in another window, I went into Snipe-IT and created an admin group, I added the admin group to the AD user in Snipe-It. Went back to my incognito window and refreshed. The user had admin permissions.

Try to give a user all privileges and see if that works. Also, _target_id=161_ is 161 the id of the AD user that you gave the permissions to?

**Comment by elreydetoda** [comment]
So I found out what happened, so when I added the permissions to the users. I was bulk editing the people, and was adding them to groups through the bulk edit option. Which I found out, that (the bulk edit) is the actual problem, there is no ldap problem at all. 

So when I bulk edit and add the permissions that way, it isn't assigning the permissions properly. I can just go to a user manually not change anything and just hit save, and afterwards the person it perfectly fine. They have the proper permissions. So I don't know what it is, but even though the user is getting labeled properly (with the proper group) in the bulk edit. I guess it isn't actually assigning the permission properly (from wherever it keeps track of that, guessing it is in the database).
